docs/website/mkdocs.yml

```diff
@@ -27,6 +27,10 @@ theme:
 
     - navigation.indexes # section names can link to index.md pages
 
+    - content.code.annotate # Allow inline annotations
+    - content.code.copy # Enable copy button
+    - content.tabs.link # Link content tabs across site (e.g. Windows/Linux)
```

Comment:
I'm also
* turning on the copy button for code blocks, which unfortunately doesn't strip comments
* Enabling code block annotations, but not adding any. I tried to add a single annotation shared across Linux/macOS/WIndows content tabs but that didn't work

Reply:
(The copy button was enabled before, but it is now opt-in. We aren't currently pinning the version of mkdocs-material that we download, so we lost the feature in ~December)